docs/CSS28-migration_guide.md

```diff
@@ -0,0 +1,144 @@
+
+## Improvement of overflow
+
+Version 2 ships with a couple of improvements specific to `overflow`.
+
+In version 1, some longer strings the rendering engine could not break and wrap would be visibly overflowing in the next column. In version 2, this is now clipped/hidden at the `body` level, in line with what other Reading Systems have been doing to mitigate this issue. 
+
+As a consequence, you should check whether this impacts progression within XHTML documents, depending on your implementation of gestures, taps, etc.
+
+If you already solved this issue by adding some styles for `overflow`, you may also have to remove these styles if possible, or report possible issues and malfunctions so that ReadiumCSS can improve `overflow` management more reliably.
+
+## Re-implementation of the font-size user setting
+
+In version 1, ReadiumCSS had to rely on the `:root`’s `font-size` and the cascade. In order to make it reliable, a font-normalization patch was used to get around author stylesheets’ using absolute values like `px` or `pt`. This created important side effects and their resolution was long overdue.
+
+In version 2, ReadiumCSS switched to `zoom`, which makes the patch no longer needed, except for rendering engines/browsers that don’t support this CSS property.
+
+It doesn’t need any change at the implementation level, and should work out of the box. All is handled behind the scenes in ReadiumCSS font-size module.
+
+## Extension of the font-family user setting override
+
+In version 1, the font-family user setting overrode a selection of elements of body copy e.g. `p`, `li`, `dt`, etc. This explained why the font-family of headings would remain the same as the one set in authors’ stylesheets for instance.
+
+In version 2, the logic has changed and the override will now apply to everything except a handful of tags e.g. `code`, `var`, `samp`, and `kbd`. It is actually borrowed from the accessibility fonts and normalization module.
+
+## Updated default font stacks
+
+The default font stacks for latin in `ReadiumCSS-base` module have been updated to benefit from newer fonts on the Windows platform. They have also been extended to offer better coverage for Linux distributions, with project [modern font stacks](https://modernfontstacks.com) as a reference.
+
+## Updated recommandations for libre and open source fonts
+
+Typeface Clear Sans has been replaced with IBM Plex Sans, the former project having been archived, with no more design and development happening in the future.
+
+Luciole Vision and Atkinson Hyperlegible have been added to the list of recommended fonts for accessibility. Due to its unknown copyright, font [Maqroo](https://maqroo.com) couldn’t be added to this list for arabic.
+
+## Support for variable fonts
+
+In version 2, ReadiumCSS provides support for three registered (read common) axes so that you can expose them as user settings. 
+
+1. `--USER__fontWeight`
```

Comment:
It’s using `font-weight` so yeah it should work with all fonts, although with some you may be limited to 2 weight e.g. `400` and `700`. Others have more weights available so it would work almost as well as variable fonts, although with visible steps in the range.